### PR TITLE
W-13667358: Add config.toml and parameterize registry url. Add setup …

### PR DESCRIPTION
**Pull request by balbifm**
…goal in Makefile

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -7,7 +7,7 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-pdk = { path = ".pdk/pdk/pdk" }
+pdk = { version = "0.1.0", registry = "anypoint" }
```
Comment (balbifm) [inline Cargo.toml:10]: Should we parameterize the version to be populated in scaffolding?

File: Makefile [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
-DEFINITION_GCL_PATH := $(shell anypoint-cli-v4 pdk policy-project locate-gcl definition)
-ASSET_VERSION := $(shell cargo anypoint get-version)
+TARGET         		:= wasm32-wasi
+TARGET_DIR     		:= target/$(TARGET)/release
+NAME           		:= {{ crate_name }}
+CARGO_ANYPOINT 		:= cargo-anypoint
+DEFINITION_NAME     = $(shell anypoint-cli-pdk-plugin pdk policy-project definition get gcl-metadata-name)
+DEFINITION_GCL_PATH = $(shell anypoint-cli-v4 pdk policy-project locate-gcl definition)
+ASSET_VERSION       = $(shell cargo anypoint get-version)
+OAUTH_TOKEN         = $(shell anypoint-cli-v4 conf token | grep -o '"token": *"[^"]*"' | cut -d '"' -f 4)
```
Comment (balbifm) [inline Makefile:8]: This won't work in Windows. I'm leaning on adding a command in the CLI anyway..

Comment (nledesma) [reply Makefile:8]: agree! but we can add it in the future

File: Cargo.toml (hunk 1 above)
Comment (nledesma) [reply Cargo.toml:10]: it depends. maybe we want this template to be strictly related to one particular version of the PDK. and handle the template+pdk versioning somehow like chained dependencies.

Comment (balbifm) [reply Cargo.toml:10]: Yes, that is the related other open question. Should we tag the template each time we release the PDK (and there is a change in the template)? Does cargo generate supports reading from a tag?

**Pull request merged**